activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-jpa/src/main/java/org/activiti/cloud/services/modeling/entity/ProjectConfigurationEntity.java

```diff
@@ -52,7 +55,12 @@ public ProjectConfigurationEntity() {  // for JPA
     }
 
     public ProjectConfigurationEntity(Boolean enableCandidateStarters) {
-        this.enableCandidateStarters = enableCandidateStarters;
+        this.enableCandidateStarters = Optional.of(enableCandidateStarters).orElse(false);
```

Comment:
Isn't it better to use primitive type boolean enableCandidateStarters so that it's always false by default?